index.html

```diff
@@ -3465,13 +3465,13 @@ <h2>JSON-LD Extensibility</h2>
             <li>
 Extending the machine-readable vocabularies used to describe information in the
 data model, without relying on a centralized system, is accomplished through
-the use of [[LINKED-DATA]].
+the use of [[?LINKED-DATA]].
             </li>
 
             <li>
 Support for multiple types of cryptographic proof formats is accomplished
-through the use of Linked Data Proofs [[LD-PROOFS]], Linked Data Signatures
-[[LD-SIGNATURES]], and a variety of signature suites.
+through the use of Linked Data Proofs [[?LD-PROOFS]], Linked Data Signatures
+[[?LD-SIGNATURES]], and a variety of signature suites.
```

Comment:
I believe this is being archived and moved into the LD-Proofs work in the CCG. See here for details: https://github.com/w3c-ccg/ld-signatures/issues/29#issuecomment-590979626

Reply:
Updated, cheers @kdenhartog